src/jvmMain/kotlin/utils/tokenStorage.kt

```diff
@@ -34,9 +35,12 @@ actual fun tokenStorageWrite(data: String) {
 
 actual fun tokenStorageGet(): String {
 
-    val fileContent = Files.readAllLines(Path.of(getAbsolutePath()))
-
-    return fileContent[2]
+    try {
+        val fileContent = Files.readAllLines(Path.of(getAbsolutePath()))
+        return fileContent[2]
+    } catch (e: IOException) {
+        return ""
```

Comment:
Let's be sure to add documentation on this so we know it returns an empty string if the token couldn't be parsed.